content/modeling/modeling_4.md

```diff
@@ -0,0 +1,121 @@
++++
+title= "Create Fractribution report table"
+weight = 4
+post = ""
++++
+
+Data from the fractribution dbt package should now be in your warehouse. Next we need to create the report table. There are two ways to to this - by pulling a Docker container image or with Python locally. 
+***
+
+{{< tabs groupId="modeling" >}}
+{{% tab name="Docker" %}}
+To run the fractribution script using Docker, make sure that you have [Docker](https://www.docker.com/products/docker-desktop/) installed.
+
+Once you have docker installed, you can pull the image from the Docker Hub:
+
+```
+docker pull snowplow/fractribution
+```
+
+***
+You can optionally choose a specific attribution model from the following:
+- shapley (default): Takes the weighted average of the marginal contributions of each channel to a conversion.
```

Comment:
Could we maybe link to a credible resource on how this works ?
E.g. https://support.google.com/analytics/answer/3191594?hl=en#algorithm _(if we approve it)_

Reply:
Sure. This could also be useful to link to somewhere? https://github.com/google/fractribution/blob/master/Fractribution_Slides.pdf

Reply:
I have linked to both of these, we can remove if we change our minds